src/stores/app/index.js

```diff
@@ -257,26 +286,40 @@ export default class AppStore extends Store {
   };
 
   @action
-  modify = async (params = {}) => {
+  modify = async (params = {}, hasNote) => {
     this.isLoading = true;
     this.createResult = await this.request.patch('apps', params);
     this.isLoading = false;
 
-    if (get(this.createResult, 'app_id')) {
+    if (hasNote && get(this.createResult, 'app_id')) {
       this.info('应用信息保存成功');
     }
   };
 
   @action
   modifyApp = async e => {
-    e.preventDefault();
+    if (e) {
+      e.preventDefault();
+    }
+
+    const data = _.pick(this.appDetail, [
+      'name',
+      'abstraction',
+      'description',
+      'home',
+      'icon'
+    ]);
 
-    const data = getFormData(e.target);
     await this.modify(
-      _.assign(data, {
-        app_id: this.appDetail.app_id,
-        category_id: this.appDetail.category_id
-      })
+      _.assign(
+        data,
+        {
+          app_id: this.appDetail.app_id,
+          category_id: this.appDetail.category_id,
+          icon: this.appDetail.icon
+        },
+        Boolean(e)
```

Comment:
what does this `Boolean(e)`?  e is event ?

Reply:
e is event ,transfer this  param for the Stepper Component not used notification.